src/core/agent/agent.ts

```diff
@@ -328,6 +333,63 @@ class Agent {
     });
   }
 
+  async devPreload() {
+    try {
+      await SecureStorage.get(KeyStoreKeys.APP_PASSCODE);
+    } catch (error) {
+      if (
+        error instanceof Error &&
+        error.message ===
+          `${SecureStorage.KEY_NOT_FOUND} ${KeyStoreKeys.APP_PASSCODE}`
+      ) {
+        await SecureStorage.set(
+          KeyStoreKeys.APP_PASSCODE,
+          Agent.APP_PASSSCODE_DEV_MODE
+        );
+      }
+    }
+
+    try {
+      await SecureStorage.get(KeyStoreKeys.SIGNIFY_BRAN);
+    } catch (error) {
+      if (
+        error instanceof Error &&
+        error.message ===
+          `${SecureStorage.KEY_NOT_FOUND} ${KeyStoreKeys.SIGNIFY_BRAN}`
+      ) {
+        const branAndMnemonic = await Agent.agent.getBranAndMnemonic();
```

Comment:
`this.getBranAndMnemonic` - if we don't stop with `Agent.agent` we'll have to remove the singleton completely and let the UI initialise it.

Reply:
Anyway, we don't need the mnemonic. Just use `randomPasscode().substring(0, 21)`